### ISSUE TITLE
Local media doesn't play on the apk v27

### ISSUE THREAD

**Issue by nightsh** [rootio/rootio_handset#37]
Playlists that have a start time in the past won't start. If I schedule new playlists in the future, they won't get picked up either.

I am getting a "TIme issue..." message every now and then.

According to the code, the app checks if:

* program is local (music / stream)
* program starts in the future
* if these two conditions are met, it gets scheduled

In our case, we have a music program starting at 6:00 AM and ending at 5:00 PM. So starting the app anywhere in between these hours will result in the program not being played.

Not sure what to think about the newly scheduled music programs, they shold've been picked up by the app by they weren't.

If there's anything I can add here to further debug this, let me know.

**Comment by csik** [comment]
This is true for my phone as well. If I restart services, however, it will start the playlist.

I am also getting the "Time issue..." message.

**Comment by judelove** [comment]
A fresh install of version 27 fixes this issue. 

However on a fresh installation, the phone needs ~20 minutes to pull down schedules upto the current day/hour. 

Starting from 27 onwards all changes will not need re-installation.
